lib/util/ids.js

```diff
@@ -0,0 +1,6 @@
+var i  = 0
+module.exports = function (returnVal) {
+  if (returnVal) return 'js-ids-' + i
```

Comment:
this option will not be used any more I guess. We can keep it and remove it as well.